src/indev/lv_indev.c

```diff
@@ -1267,6 +1246,23 @@ static void indev_proc_press(lv_indev_t * indev)
  */
 static void indev_proc_release(lv_indev_t * indev)
 {
+    if((indev->pointer.last_point.x != indev->pointer.act_point.x ||
+        indev->pointer.last_point.y != indev->pointer.act_point.y)) {
+        lv_obj_t ** layer = &indev->disp->sys_layer;
+        lv_obj_t ** last = &indev->pointer.last_hovered;
+        lv_obj_t * hved = NULL;
+        uint8_t layers = 4;
+        while(!hved && layers) {
```

Comment:
Relying on the order of field in a struct is not safe and can result in hard to debug issues. It seems [`pointer_search_obj`](https://github.com/lvgl/lvgl/blob/master/src/indev/lv_indev.c#L1388) can be used here.